src/Sylius/Bundle/AdminBundle/Resources/config/routing/partial/order.yml

```diff
@@ -7,3 +7,27 @@ sylius_admin_partial_order_latest:
                 method: findLatest
                 arguments: [$count]
             template: $template
+
+sylius_admin_partial_order_shipment_ship:
+    path: /orders/{id}/shipments/{shipmentId}/ship
+    methods: [GET, PUT]
+    defaults:
+        _controller: sylius.controller.shipment:updateAction
+        _sylius:
+            repository:
+                method: findByOrderIdAndId
+                arguments:
+                    orderId: $id
+                    id: $shipmentId
```

Comment:
Actually I think the routing should be `/orders/{orderId}/shipments/{id}/ship` we have this convention for promotion coupons, variants etc. and the then arguments would be `orderId: $orderId, id: $id`.

This should be also updated for the payments.